### PR TITLE
Issue #3681 - Documentation: front matter directives and improve anax…

### LINKED ISSUE #3681: Documentation: Improve the anax-in-container documentation
### What is the current documentation state?

Improve the anax-in-container documentation.  This is a continuation of #3636 and [#3680](https://github.com/open-horizon/anax/pull/3680)

Reformat and describe the various options.

https://github.com/open-horizon/anax/blob/master/docs/agent_container_manual_deploy.md

### Where is this stated?

_No response_

### Why do you want to improve the statement?

_No response_

### Proposed Statement

_No response_

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by johnwalicki**
…-in-container instructions

# Pull Request Template

## Description

Fixes #3681 
- Add front matter liquid directives for codeblocks
- whitespace around commands
- improve the anax-in-container documentation

## Type of change

- [x] This change requires a documentation update

## How Has This Been Tested?

Visit the rendered pages at https://open-horizon.github.io/docs/anax/docs/

## Checklist:

- [x] I have made corresponding changes to the documentation
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: docs/agent_container_manual_deploy.md [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
 ## Starting an agent in a container on Linux
 {: #container-agent}
 
-These instructions assume that the agent container you want to deploy already exists and is in a container image repository.
+There are several techniques to start the agent in a container:
 
-### Usage
-{: #container-usage}
+* Option 1 -The [agent-install instructions ](https://github.com/open-horizon/anax/tree/master/agent-install){:target="_blank"}{: .externalLink} provide a `--container` option to download and provision the agent in a container.
+* Option 2 - If you have already installed the Horizon CLI package, it installed a `horizon-container` script onto your system.
```
Comment (dlarson04) [inline docs/agent_container_manual_deploy.md:26]: how about 
```
If you have already installed the Horizon CLI package, it installed a `horizon-container` script onto your system that can be used to start the anax in container
```

**Review by dlarson04 — CHANGES_REQUESTED** [review]

File: docs/agent_container_manual_deploy.md [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+
 ## Starting an agent in a container on Linux
 {: #container-agent}
 
-These instructions assume that the agent container you want to deploy already exists and is in a container image repository.
+There are several techniques to start the agent in a container:
 
-### Usage
-{: #container-usage}
+* Option 1 -The [agent-install instructions ](https://github.com/open-horizon/anax/tree/master/agent-install){:target="_blank"}{: .externalLink} provide a `--container` option to download and provision the agent in a container.
```
[the remark this answers is not in the document]
Comment (johnwalicki) [reply docs/agent_container_manual_deploy.md:25]: fixed

File: docs/agent_container_manual_deploy.md [hunk 3]
```diff
@@ -1,29 +1,36 @@
...
+{:codeblock: .codeblock}
+{:pre: .pre}
+{:child: .link .ulchildlink}
+{:childlinks: .ullinks}
+
 ## Starting an agent in a container on Linux
 {: #container-agent}
 
-These instructions assume that the agent container you want to deploy already exists and is in a container image repository.
+There are several techniques to start the agent in a container:
```
[the remark this answers is not in the document]
Comment (johnwalicki) [reply docs/agent_container_manual_deploy.md:23]: good - fixed.

File: docs/agent_container_manual_deploy.md (hunk 1 above)
Comment (johnwalicki) [reply docs/agent_container_manual_deploy.md:26]: fixed.

**Review by dlarson04 — APPROVED** [review]

**Pull request merged**